packages/grafana-runtime/src/config.ts

```diff
@@ -107,7 +109,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
   pluginAdminExternalManageEnabled = false;
   pluginCatalogHiddenPlugins: string[] = [];
   expressionsEnabled = false;
-  customTheme?: any;
+  customTheme?: undefined;
```

Comment:
```suggestion
  customTheme? = undefined;
```
`customTheme?` means that it can be undefined, so setting it as a type is redundant. Not sure about this change in general, it seems like `any` might be ok here.

Reply:
it was causing betterer precomit to fail :shrug: so this was the only way I could get it to pass